sdv/constraints/base.py

```diff
@@ -226,6 +227,14 @@ def _validate_constraint_columns(self, table_data):
         """
         missing_columns = [col for col in self.constraint_columns if col not in table_data.columns]
         if missing_columns:
+            if not self._columns_model:
```

Comment:
@csala I wasn't sure if this is the correct scenario to warn for. Could you elaborate on why missing columns might cause the reject sampling to become slow?

Reply:
@katxiao I submitted issue #550 for completion, so the change and its motivation is actually registered on the issues side. I included there a small description of why conditional sampling will become slower if the columns model is not fitted.